images/capi/packer/config/cni.json

```diff
@@ -1,7 +1,7 @@
 {
-    "kubernetes_cni_semver": "v0.7.5",
-    "kubernetes_cni_rpm_version": "0.7.5-0",
-    "kubernetes_cni_deb_version": "0.7.5-00",
+    "kubernetes_cni_semver": "v0.8.6",
+    "kubernetes_cni_rpm_version": "0.8.6-0",
+    "kubernetes_cni_deb_version": "0.8.6-00",
```

Comment:
compared to https://github.com/kubernetes-sigs/image-builder/pull/258/files. not all changes are pulled here. can u please confirm? 
Do you have a chance to verify this for both latest version and non-latest version, like 1.16.11/1.16.10 on azure and other clouds? 
Also, how about this comment:  https://github.com/kubernetes-sigs/image-builder/pull/258#issuecomment-648946072. I don't quite understand it.

Reply:
Few changes are default k8s version in image-builder, as earlier it was 1.16.2 and now they have changed 1.16.11.
I did not test the change will verify and let you know.
As per comment, it seems the only mentioned version is supported for 0.8.6.
Let me check other versions also with it and update you accordingly.